From 3f74504f17f3bb36d76c43ec6cfcea304ced8ab0 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:03:51 -0400
Subject: [powerpc] pseries: Use 'true' instead of '1' for orderly_poweroff

Message-id: <1410545655-205645-3-git-send-email-dzickus@redhat.com>
Patchwork-id: 94066
O-Subject: [RHEL7 PATCH 002/626] powerpc/pseries: Use 'true' instead of '1' for orderly_poweroff
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 1b7e0cbe4969dd4c2945b2fb8d6881a03b4d0ac7
Author: liguang <lig.fnst@cn.fujitsu.com>
Date:   Thu May 30 15:20:33 2013 +0800

    powerpc/pseries: Use 'true' instead of '1' for orderly_poweroff

    orderly_poweroff is expecting a bool parameter, so
    use 'true' instead '1'

    Signed-off-by: liguang <lig.fnst@cn.fujitsu.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/platforms/pseries/ras.c b/arch/powerpc/platforms/pseries/ras.c
index c4dfccd..7b3cbde 100644
--- a/arch/powerpc/platforms/pseries/ras.c
+++ b/arch/powerpc/platforms/pseries/ras.c
@@ -83,7 +83,7 @@ static void handle_system_shutdown(char event_modifier)
  switch (event_modifier) {
  case EPOW_SHUTDOWN_NORMAL:
   pr_emerg("Firmware initiated power off");
-  orderly_poweroff(1);
+  orderly_poweroff(true);
   break;
 
  case EPOW_SHUTDOWN_ON_UPS:
@@ -95,13 +95,13 @@ static void handle_system_shutdown(char event_modifier)
   pr_emerg("Loss of system critical functions reported by "
    "firmware");
   pr_emerg("Check RTAS error log for details");
-  orderly_poweroff(1);
+  orderly_poweroff(true);
   break;
 
  case EPOW_SHUTDOWN_AMBIENT_TEMPERATURE_TOO_HIGH:
   pr_emerg("Ambient temperature too high reported by firmware");
   pr_emerg("Check RTAS error log for details");
-  orderly_poweroff(1);
+  orderly_poweroff(true);
   break;
 
  default:
@@ -162,7 +162,7 @@ void rtas_parse_epow_errlog(struct rtas_error_log *log)
 
  case EPOW_SYSTEM_HALT:
   pr_emerg("Firmware initiated power off");
-  orderly_poweroff(1);
+  orderly_poweroff(true);
   break;
 
  case EPOW_MAIN_ENCLOSURE:
-- 
1.7.1